README.md

```diff
@@ -5,14 +5,22 @@
 [![Build Status][ci-img]][ci]
 [![Report Card][report-card-img]][report-card]
 
-## Status
+UberFx is a flexible, modularized framework for building robust and performant
+services. It removes boilerplate code out of your hands and lets you focus on
+your application logic.
 
-Pre-ALPHA. API Changes are **highly likely**.
+## Status
 
-## Abstract
+Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
 
-This framework is a flexible, modularized basis for building robust and
-performant services at Uber with the minimum amount of developer code.
+## What's included
```

Comment:
This section is wonderful.

Something to think about: UberFx doesn't just _include_ these things. It actually gets them all started for you and plops them into your project. That's awesome.

For people who haven't used this, do we want stronger language, like "What's included" as a heading, with a sentence below, like "UberFx builds the following into your new service:" to be extra clear?